index.html

```diff
@@ -34,6 +34,167 @@ <h1 class="header-h1">
         </button>
       </div>
     </header>
+    <div class="projects-cont">
```

Comment:
- [ ] Kindly make to wrap your project cards into the `section` tag in order to use format our HTML  semantically which will increase your website accessibility. :wink: